mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -365,6 +366,80 @@ func TestAccResourceMongoDBAtlasCluster_AWSWithLabels(t *testing.T) {
 	})
 }
 
+func TestAccResourceMongoDBAtlasCluster_withPrivateEndpointLink(t *testing.T) {
+	var cluster matlas.Cluster
+
+	resourceName := "mongodbatlas_cluster.test"
+
+	awsAccessKey := os.Getenv("AWS_ACCESS_KEY_ID")
+	awsSecretKey := os.Getenv("AWS_SECRET_ACCESS_KEY")
+
+	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID")
+	region := os.Getenv("AWS_REGION")
+	providerName := "AWS"
+
+	vpcID := os.Getenv("AWS_VPC_ID")
+	subnetID := os.Getenv("AWS_SUBNET_ID")
+	securityGroupID := os.Getenv("AWS_SECURITY_GROUP_ID")
+	clusterName := fmt.Sprintf("test-acc-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t); checkAwsEnv(t); checkPeeringEnvAWS(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckMongoDBAtlasClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMongoDBAtlasClusterConfigWithPrivateEndpointLink(
+					awsAccessKey, awsSecretKey, projectID, providerName, region, vpcID, subnetID, securityGroupID, clusterName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckMongoDBAtlasClusterExists(resourceName, &cluster),
+					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccResourceMongoDBAtlasCluster_withAzureNetworkPeering(t *testing.T) {
+	var peer matlas.Peer
+
+	resourceName := "mongodbatlas_network_peering.test"
+	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID")
+	directoryID := os.Getenv("AZURE_DIRECTORY_ID")
+	subcrptionID := os.Getenv("AZURE_SUBCRIPTION_ID")
+	resourceGroupName := os.Getenv("AZURE_RESOURSE_GROUP_NAME")
```

Comment:
```suggestion
	resourceGroupName := os.Getenv("AZURE_RESOURCE_GROUP_NAME")
```
noticed a small typo but fine with me if it's saved like this in the env!

Reply:
thank you, chris, I fixed it the typo

Reply:
Looks like this came back by accident